test/Strings/rlexe.xml

```diff
@@ -105,7 +105,7 @@
   <test>
     <default>
       <files>compare.js</files>
-      <baseline>compare.baseline</baseline>
+      <baseline/>
```

Comment:
If there's no baseline now, I'd prefer to remove the `<baseline/>` tag completely.

Reply:
I don't have a strong op. on that, so removed.